runtime/src/main/java/org/corfudb/runtime/clients/SequencerClient.java

```diff
@@ -73,6 +73,17 @@ public SequencerClient(IClientRouter router, long epoch) {
                 new TokenRequest(numTokens, streamIDs, conflictInfo)));
     }
 
+    /**
+     * Fetches the next counter value from the sequencer as an ordered globally unique id.
+     *
+     * @param numTokens Number of tokens to be reserved.
+     * @return A completable future with the token response from the sequencer.
+     */
+    public CompletableFuture<TokenResponse> nextOrderedGuid(long numTokens) {
```

Comment:
If we are always passing `1` for `numTokens`, why we need this parameter?

Reply:
2 reasons: 
1. I am following same patter as next() Token, so it is copy pasted convention in a way.
2. There was a discussion that we will soon need ability to request multiple tokens/ids in one request atomically. So this just provisions for that.